scripts/trigger_convergence.py

```diff
@@ -123,6 +136,35 @@ def get_groups(parsed, store, conf):
     return d
 
 
+@do
+def group_steps(group):
+    """
+    Return Effect of list of steps that would be performed on the group
+    if convergence is triggered on it with desired=actual
+    """
+    now_dt = yield Effect(Func(datetime.utcnow))
+    all_data_eff = convergence_exec_data(
+        group["tenantId"], group["groupId"], now_dt, get_executor)
+    all_data = yield Effect(TenantScope(all_data_eff, group["tenantId"]))
+    (executor, scaling_group, group_state, desired_group_state,
+     resources) = all_data
+    desired_group_state.desired = len(resources['servers'])
```

Comment:
It might raise `KeyError` from `resources["servers"]`. However, I am not concerned with that since this script is only used to transition _existing_ tenants with `launch_server` groups to convergence.